docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -1150,8 +1156,29 @@ into bytes that can be stored in the validator's Radix-Merkle tree.
 
 {% endif %}
 
+Addressing
+----------
+
 By convention, we'll store game data at an address obtained from hashing the
-game name prepended with some constant:
+game name prepended with some constant.
+
+XO data is stored in state using addresses generated from the XO
+family name and the name of the game being stored. In particular, an
+XO address consists of the first 6 characters of the SHA-512 hash of
+the UTF-8 encoding of the string "xo" (which is "5b7349") plus the
+first 64 characters of the SHA-512 hash of the UTF-8 encoding of the
+game name.
+
+For example, the XO address for a game called "mygame" could be
+generated as follows:
+
+.. code-block:: pycon
+
+    >>> hashlib.sha512('xo'.encode('utf-8')).hexdigest()[:6] +
+    >>> hashlib.sha512('mygame'.encode('utf-8')).hexdigest()[:64]
```

Comment:
nit, but if you make other changes, you use "my-game" in #1466 
If you update that here your hash will be ... 
```>>> x = hashlib.sha512('xo'.encode('utf-8')).hexdigest()[:6]
>>> x
'5b7349'
>>> y = hashlib.sha512('my-game'.encode('utf-8')).hexdigest()[:64]
>>> y
'4d4cffe9cf3fb4e41def5114a323e292af9b0e07925cca6299d671ce7fc7ec37'
>>> x+y
'5b73494d4cffe9cf3fb4e41def5114a323e292af9b0e07925cca6299d671ce7fc7ec37'
```

Reply:
THANKS!!!!